exercise-1.js

```diff
@@ -130,7 +207,11 @@
         //use javascript to compute the value of the above statement. Each individual operation needs to be a function expression. run all the functions after defining them, and print the answer to the console.
 
         //your code...
-
+        function equation (){
```

Comment:
The exercise is asking to do individual operations. Each operation in one function.